.github/workflows/vitest.yml

```
@@ -0,0 +1,19 @@
+name: 'Tests: vitest'
+
+on:
+  push:
+    branches: [ "main", "ragezbla/89-vitest-gh-actions" ] # TODO remove after checking it works
+  pull_request:
+    # The branches below must be a subset of the branches above
+    branches: [ "main" ]
+  schedule:
+    - cron: '45 15 * * 4'
+
+jobs:
+  cypress-run:
+    runs-on: ubuntu-latest
```

Add a name section for it to be more user friend in the GitHub PR view

```suggestion
    runs-on: ubuntu-latest
    name: Running Pinia Tests 🚀
```